mappings/net/minecraft/command/BaseCommand.mapping

```diff
@@ -0,0 +1,54 @@
+CLASS net/minecraft/class_1999 net/minecraft/command/BaseCommand
```

Comment:
I think `BaseCommand` might be misleading, on the other hand there aren't much names i can think of except `AbstractCommand` or `CommandBase`